functions.php

```diff
@@ -38,9 +38,9 @@ function register_my_menu() {
 add_action( 'init', 'register_my_menu' );
 ?>
 
-<?php
+<!--<?php
```

Comment:
you're putting php in html comment tags, that's not going to work. Check http://www.tizag.com/phpT/comment.php